nixos/modules/services/desktops/wlock.nix

```diff
@@ -0,0 +1,24 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.wlock;
+in
+{
+  options = {
```

Comment:
Please add services.wlock.package option. There are many existing examples in nixpkgs already.